packages/react-components/src/components/TextFieldWithMention/TextFieldWithMention.tsx

```diff
@@ -562,9 +562,8 @@ export const TextFieldWithMention = (props: TextFieldWithMentionProps): JSX.Elem
         // Update the caret position, used for positioning the suggestions popover
         const textField = event.currentTarget;
         const relativePosition = Caret.getRelativePosition(textField);
-        const adjustOffset = Math.max(0, textField.scrollHeight - textField.clientHeight);
```

Comment:
|after scrolling and typing the position looks okay|when open the page for the first time, paste text and start to add a first mention, the mention popover is a little bit higher|
|-|-|
| <img width="607" alt="image" src="https://github.com/Azure/communication-ui-library/assets/98852890/e247d731-81ee-43b9-8246-1ff09971b461">|<img width="536" alt="image" src="https://github.com/Azure/communication-ui-library/assets/98852890/e869b5ac-1ac6-4063-80c5-bedb672d1c92"> |